### PR TITLE
[PLT-5440] Add Options to System Console Dropdown Menu

### PR DESCRIPTION
**Pull request by R-Wang97**
#### Summary
Add some additional options to the dropdown menu in System Console. Options added include:
```
Administrator's Guide - External Link
Troubleshooting Forum - External Link
Commercial Support - External Link
About Mattermost - Opens About Build Modal
```
External links open in a new tab.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-5440
https://github.com/mattermost/platform/issues/5355

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks @R-Wang97! 

@lindalumitchell please help with initial testing?

File: config/config.json [hunk 1]
```diff
@@ -153,6 +153,9 @@
         "AboutLink": "https://about.mattermost.com/default-about/",
         "HelpLink": "https://about.mattermost.com/default-help/",
         "ReportAProblemLink": "https://about.mattermost.com/default-report-a-problem/",
+        "AdministratorsGuideLink": "https://about.mattermost.com/administrators-guide/",
```
Comment (jasonblais) [inline config/config.json]: @R-Wang97 if these are blank, do the options disappear from the System Console main menu?

Comment (R-Wang97) [inline config/config.json]: I believe they shouldn't, because they're also hardcoded in `model/config.go`, which is what's used if it can't be found in `config.json`

File: webapp/components/admin_console/admin_navbar_dropdown.jsx [hunk 2]
```diff
@@ -45,6 +49,26 @@ export default class AdminNavbarDropdown extends React.Component {
...
     }
 
+    componentDidUpdate() {
+        if (document.body.classList.contains('app__body') && this.state.showAboutModal === false) {
+            // Timeout needed to ensure modal closing animation can fully complete
+            setTimeout(() => {
+                $('body').removeClass('app__body');
+            }, 300);
+        }
+    }
```
Comment (jwilander) [inline webapp/components/admin_console/admin_navbar_dropdown.jsx]: If you'd like to preserve the animation, instead of doing it this way can you convert the modal to close [like this one](https://github.com/mattermost/platform/blob/master/webapp/components/edit_channel_header_modal.jsx)? You can then use `ToggleModalButton` [similar to here](https://github.com/mattermost/platform/blob/master/webapp/components/channel_header.jsx#L372) which should preserve the animation

Also just a heads-up [we're moving away from using jQuery](https://docs.mattermost.com/developer/core-developer-handbook.html#technical-direction-resolutions)

Comment (R-Wang97) [inline webapp/components/admin_console/admin_navbar_dropdown.jsx]: I used jQuery so that I could add the `app__body` class, so that the modal would render with the correct style. Should I remove the jQuery and try modifying the sass so that it doesn't need that class to render properly?

Comment (jwilander) [inline webapp/components/admin_console/admin_navbar_dropdown.jsx]: Yes, please do that

**Comment by jwilander** [comment]
@asaadmahmood can you take a look at the CSS changes?

**Comment by asaadmahmood** [comment]
@R-Wang97 Why did you create a mixin for the `about modal`? I don't think we're going to use it in multiple places.

**Comment by R-Wang97** [comment]
@asaadmahmood I created a mixin because with this PR, there are now two places where it can be accessed, from the menu dropdown in a user view, as well as in a dropdown from system console. In system console, there's no `app__body` CSS class, so I originally used jQuery to add that class, but @jwilander said not to use jQuery, so I came up with the idea of using a mixin.

**Comment by asaadmahmood** [comment]
Which is fine, we can just have the `.about-modal` without the `.app__body`, we don't need to create a mixin since the mixin itself is not being reused in the css.

**Comment by asaadmahmood** [comment]
I say that because with what we have now, the modal classes atleast on the main view with `.app__body`, will be needlessly duplicated twice.

**Comment by R-Wang97** [comment]
If I remove that block of code, the about modal renders with incorrect styling on the main view. 
screen shot 2017-04-24 at 8 58 45 pm

**Comment by asaadmahmood** [comment]
Can you also post a screenshot of it rendering with that block of quote?

**Comment by jasonblais** [comment]
@R-Wang97 Sorry Ryan for jumping in here late.

I've been thinking about the config.json settings more and I don't think they're necessary. I don't see a use case where the admin would want to change the link for `Commercial Support` for instance, and thus it would be better if it was hard-coded without a configurable setting. I think that was also the original intent of the Help Wanted ticket: https://github.com/mattermost/platform/issues/5355

The "Help" and "Report a Problem" links that appear in the team Main Menu are different, as some companies have their own help links or separate systems to report issues (some customers use this). 

However, the links added here only appear for the System Admins, which is <<1% of the user base.

In summary: Propose removing the options from `config.json`. Otherwise no changes required. Really sorry for a last-minute change,

**Comment by R-Wang97** [comment]
@jasonblais No worries, so I'll revert my changes in `config.json`?

**Comment by jasonblais** [comment]
Yes please, I'm not sure if anything else needs to be changed?

And thanks @R-Wang97, appreciate it
